### PR TITLE
Fix test order (4) - Set/reset globals

### PR DESCRIPTION
**Pull request by m-vo**
| Q                | A
| -----------------| ---
| Fixed issues     | partly solves #2316 
| Docs PR or issue | -

I'm working on test isolation. This part is about handling global variables.

This PR adds/sets...
1) missing `unset()` statements for global variables,
2) process isolation where needed,
3) debug mode to `false` when comparing templates.

### CODE REVIEW

File: manager-bundle/tests/HttpKernel/ContaoKernelTest.php [hunk 1]
```diff
@@ -102,6 +102,9 @@ public function testRegisterBundles(): void
         $this->assertArrayNotHasKey(AppBundle::class, $bundles);
     }
 
+    /**
+     * @runInSeparateProcess
```
Comment (leofeyer) [inline manager-bundle/tests/HttpKernel/ContaoKernelTest.php:106]: Is this really necessary?

Comment (m-vo) [reply manager-bundle/tests/HttpKernel/ContaoKernelTest.php:106]: Yes, because including the `AppBundle` file affects global state (IIRC).

Comment (leofeyer) [reply manager-bundle/tests/HttpKernel/ContaoKernelTest.php:106]: It is because of the `include`.

**Review by ausi — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Pull request merged**